### PR TITLE
{chem}[foss/2018a,gimkl/2017a,intel/2018a] WIEN2k v17.1 (REVIEW)

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

needs https://github.com/easybuilders/easybuild-easyblocks/pull/1405

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/3888a9b39c8633e64c28c259a6a62dbf for a full test report.

**Comment by migueldiascosta** [comment]
easybuilders/easybuild-easyblocks#1405 is still WIP (also, forgot the tests from the older easyconfigs, will bring them back)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/f4fbceee3d75cde490ee93787f5a664d for a full test report.

File: easybuild/easyconfigs/e/ELPA/ELPA-2015.02.002-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+version = '2015.02.002'
+
+homepage = 'http://elpa.rzg.mpg.de'
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/e/ELPA/ELPA-2015.02.002-foss-2018a.eb]: @migueldiascosta Missing `SHA256` checksum

File: easybuild/easyconfigs/e/ELPA/ELPA-2015.02.002-gimkl-2017a.eb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+version = '2015.02.002'
+
+homepage = 'http://elpa.rzg.mpg.de'
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'gimkl', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/e/ELPA/ELPA-2015.02.002-gimkl-2017a.eb]: @migueldiascosta Missing `SHA256` checksum

File: easybuild/easyconfigs/e/ELPA/ELPA-2015.02.002-intel-2018a.eb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+version = '2015.02.002'
+
+homepage = 'http://elpa.rzg.mpg.de'
+description = """Eigenvalue SoLvers for Petaflop-Applications ."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://elpa.mpcdf.mpg.de/html/Releases/%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/e/ELPA/ELPA-2015.02.002-intel-2018a.eb]: @migueldiascosta Missing `SHA256` checksum

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2058.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/8d624cb3f6bc8aa8c033d8de0db94684 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2604.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/061ac6cf85c2a42fce5ab4e909520b2e for a full test report.

**Comment by boegel** [comment]
I can't test the WIEN2k easyconfigs since I don't have access to the required sources, but I'll trust @migueldiascosta's test reports for that part of this PR...

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**